docs/api.md

```diff
@@ -10,6 +10,23 @@ nav_order: 3
 
 ## Class methods
 
+### .config
+
+
+
+### .config=(value)
+
+Sets the attribute config
+
+### .sidecar_files(extensions)
+
+Find sidecar files for the given extensions.
+
+The provided array of extensions is expected to contain
+strings starting without the "dot", example: `["erb", "haml"]`.
```

Comment:
This could be fixed in [this file](https://github.com/ViewComponent/view_component/blob/22dbe8f648b44cc73d9663082a3e5024345b28ee/lib/view_component/base.rb#L416) – I don't think the word "dot" needs to be in quotes.